.github/CODEOWNERS

```diff
@@ -63,6 +63,8 @@ src/EventsProcessor @bitwarden/team-admin-console-dev
 **/Billing @bitwarden/team-billing-dev
 src/Admin/Controllers/ToolsController.cs @bitwarden/team-billing-dev
 src/Admin/Views/Tools @bitwarden/team-billing-dev
+src/Admin/AdminConsole/Models/ProviderViewModel.cs @bitwarden/team-billing-dev
+src/Admin/AdminConsole/Views/Providers/ProviderPlans.cshtml @bitwarden/team-billing-dev
```

Comment:
I meant, you could move `ProviderPlanViewModel.cs` into a `src/Admin/Billing` folder. I think trying to specify individual files in CODEOWNERS gets messy fast and is difficult to maintain.

`ProviderViewModel` should just stay where it is I think.